src/ElvSpace.js

```diff
@@ -106,6 +106,13 @@ class ElvSpace {
       });
       elvAccount.SetAccountTenantContractAddress(tenant.address);
 
+      // We must sign in using the new account to set the groups' metadata - metadata can only be set by the groups' admin
```

Comment:
If we do not use the private key of the new account, putMeta will not succeed as it will be blocked by onlyAdmin() on meta_object.sol contract.